contrib/builder/deb/aarch64/generate.sh

```diff
@@ -97,18 +99,31 @@ for version in "${versions[@]}"; do
 			;;
 	esac
 
-	echo "# Install Go" >> "$version/Dockerfile"
-	echo "# aarch64 doesn't have official go binaries, so use the version of go installed from" >> "$version/Dockerfile"
-	echo "# the image to build go from source." >> "$version/Dockerfile"
+	case "$suite" in
+		jessie)
+			echo "# Install Go (hack: for now use pre-compiled tarball from Hypriot)" >> "$version/Dockerfile"
```

Comment:
golang in Jessie was at version 1.3.3 (according to [package tracker](https://tracker.debian.org/pkg/golang)) and it was available on arm64 (according to the [buildd logs](https://buildd.debian.org/status/package.php?p=golang&suite=jessie)).

In case you are wondering about the "package is gone" message on the package tracker post-Jessie it looks like things have split into `golang-X.Y` packages (e.g. [`golang-1.7`](https://tracker.debian.org/pkg/golang-1.7), which allows for installing multiple versions) with a [`golang-defaults`](https://tracker.debian.org/pkg/golang-defaults) source package providing the `golang` metapackage, so no more `golang` source package.